src/dmd/dmodule.d

```diff
@@ -1042,7 +1047,7 @@ extern (C++) final class Module : Package
 
         /* If it has the extension ".c", it is a "C" file.
          */
-        if (FileName.equalsExt(arg, c_ext))
+        if (FileName.equalsExt(arg, c_ext) || FileName.equalsExt(arg, i_ext))
```

Comment:
Please update the comment too.

Reply:
Comment updated,
Adding a test is WIP. I'm working through the test infrastructure of DMD and try to find the proper place for it. Is there any 'importc' releated test, yet?